samcli/commands/init/init_templates.py

```diff
@@ -204,8 +202,23 @@ def get_preprocessed_manifest(
 
         return preprocessed_manifest
 
-    def get_bundle_option(self, package_type, runtime, dependency_manager):
-        return self._init_options_from_bundle(package_type, runtime, dependency_manager)
+    def _get_manifest(self):
+        """
+        In an attempt to reduce initial wait time to achieve an interactive
+        flow <= 10sec, This method first attempts to spools just the manifest file and
+        if the manifest can't be spooled, it attempts to clone the cli template git repo or
+        use local cli template
+        """
+        try:
+            response = requests.get(self.manifest_url, timeout=10)
+            body = response.text
+        except (requests.Timeout, requests.ConnectionError):
+            self.clone_templates_repo()
```

Comment:
adding a log line could be helpful here to state that getting manifest file directly have failed and we are falling back to cloning the repository.